app/src/main/java/com/iamandiiana/newsreader/DemoApplication.java

```diff
@@ -0,0 +1,21 @@
+package com.iamandiiana.newsreader;
+
+import android.app.Application;
+
+import com.iamandiiana.data.di.RepoModule;
+
+public class DemoApplication extends Application {
```

Comment:
DemoApplication seems a bit too generic, let's rename this class to something that makes it look like part of out app, like "NewsApplication"

Reply:
Done